raiden/tests/integration/test_monitoring_service.py

```diff
@@ -0,0 +1,75 @@
+from unittest.mock import MagicMock
+
+import gevent
+import pytest
+
+from raiden.constants import MONITORING_BROADCASTING_ROOM
+from raiden.network.transport.matrix.client import Room
+from raiden.network.transport.matrix.utils import make_room_alias
+from raiden.raiden_service import update_monitoring_service_from_balance_proof
+from raiden.tests.utils.messages import make_balance_proof
+from raiden.transfer import views
+
+
+@pytest.mark.parametrize('number_of_nodes', [2])
+@pytest.mark.parametrize('number_of_tokens', [1])
+def test_monitoring_global_messages(
+        token_addresses,
+        raiden_chain,
+        retry_interval,
+        retries_before_backoff,
+        token_proxy,
+        deploy_service,
+        skip_if_not_matrix,
+):
+    app0, app1 = raiden_chain
+    token_address = token_addresses[0]
+    token_network_identifier = views.get_token_network_identifier_by_token_address(
+        views.state_from_app(app0),
+        app0.raiden.default_registry.address,
+        token_address,
+    )
+
+    """
```

Comment:
hmmm can you move the docstring to the proper place at the start of the function?